zincware/MDSuite
# introduce `save_to_db` in the calculator.py

Make the calculators easier.

- Introduce common `save_to_db` method
- Store calculator arguments in the database and load results if already run
- updated all active tests / classes

# How to Test
Running this code multiple times should only trigger the computation once.
To force another computation you can use `experiment.version += 1`
```py
result = project.run.RadialDistributionFunction()
print(result.data_dict)
```


# TODO
- ~~check that adding additional data does force a rerun~~ 69d757d
- ~~provide an argument to return all computations if data has been added (return list instead of latest)~~ not fixed
- ~~output of load, run and run with arguments that are already computed should be the same -> plot + return data_dict~~
- update all disabled classes / tests
- fix Coordination Numbers
- ~~hange species, molecules to json dumps~~
- ~~Change computation data to json dumps~~
- ~~fix RDF, ADF only having a single species~~
